standard/lexical-structure.md

```diff
@@ -1507,6 +1509,40 @@ A `#line hidden` directive has no effect on the compilation unit and line number
 
 > *Note*: Although a *PP_Compilation_Unit_Name* might contain text that looks like an escape sequence, such text is not an escape sequence; in this context a ‘`\`’ character simply designates an ordinary backslash character. *end note*
 
+### §Nullable-Directives Nullable directive
+
+The nullable directive controls the nullable context, as described below.
+
+```ANTLR
+fragment PP_Nullable
+    : PP_Whitespace? '#' PP_Whitespace? 'nullable' PP_Whitespace PP_Nullable_Action
+        (PP_Whitespace PP_Nullable_Target)? PP_New_Line
+    ;
+fragment PP_Nullable_Action
+    : 'disable'
+    | 'enable'
+    | 'restore'
+    ;
+fragment PP_Nullable_Target
+    : 'warnings'
+    | 'annotations'
+    ;
+```
+
+A nullable directive sets the denoted nullable context for subsequent lines of code, until another nullable directive overrides it, or until the end of the source code is reached. The effect of each form of nullable directive is, as follows:
+
+- `#nullable disable`: Sets the nullable context to “disabled”.
+- `#nullable enable`: Sets the nullable context to “enabled”.
```

Comment:
Needs to say both annotations and warnings are enabled.

The previous bullet could also say it disables both, though that is not so important if the  options are:

- null context disabled, or
- null context enabled with annotations and/or warnings

Reply:
Personally I'd prefer the approach of saying "enables both annotations and warnings".

Reply:
In meeting: yes, make each of these specify the effect on both flags.